src/inputs/radio/radio.jsx

```diff
@@ -437,7 +511,7 @@ class Radio extends React.Component {
 
         const rootClasses = ClassNames(
             'ui',
```

Comment:
Please use `UI_CLASS_NAME` from `./src/global/constants.js` instead of 'ui'.

Reply:
done!